action.yml

```diff
@@ -23,6 +23,10 @@ inputs:
   version-tag:
     description: 'The version tag sequence to validate. Example: "v1.2.3".'
     required: true
+  should-exist:
+    description: 'Flag to indicate if the version tag should exist in the repository. Setting to `true` disables increment validity check. Default: "false".'
```

Comment:
There is no need to write, that default value is false, since it is below.

Reply:
Fixed in commit ab6cd3a3d53c8332c19bc1ffb109bf862f5b53ff.